Competition2017/src/competition/subsystems/shooter/commands/IncreaseShooterPowerCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.shooter.commands;
+
+import org.apache.log4j.Logger;
+
+import competition.subsystems.shooter.SideShooterSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class IncreaseShooterPowerCommand extends BaseCommand {
```

Comment:
What if these two commands were combined into a "IncrementShooterPowerCommand" or similar that took a parameter for the amount?